ci-operator/step-registry/upi/install/azure/upi-install-azure-commands.sh

```diff
@@ -218,10 +220,24 @@ az network private-dns link vnet create -g $RESOURCE_GROUP -z ${CLUSTER_NAME}.${
 
 echo "Deploying 02_storage"
 VHD_BLOB_URL=$(az storage blob url --account-name $ACCOUNT_NAME --account-key $ACCOUNT_KEY -c vhd -n "rhcos.vhd" -o tsv)
-az deployment group create -g $RESOURCE_GROUP \
-  --template-file "02_storage.json" \
-  --parameters vhdBlobURL="${VHD_BLOB_URL}" \
-  --parameters baseName="$INFRA_ID"
+STORAGE_ACCOUNT_ID=$(az storage account show -g $RESOURCE_GROUP --name $ACCOUNT_NAME --query id -o tsv)
+
+# Check if it's the new template using Image Galleries instead of Managed Images
+if grep -qs "Microsoft.Compute/galleries" 02_storage.json; then
+  AZ_ARCH=$(echo $OCP_ARCH | sed 's/x86_64/x64/;s/aarch64/Arm64/')
```

Comment:
i think OCP_ARCH will be passed in as arm64 - aarch64 is only used in the context of rhcos/linux

Reply:
ok ignore this comment....you do use that to get the image from the rhcos json